### ISSUE TITLE
AtomicEditor high CPU usage on Linux

### ISSUE THREAD

**Issue by CTrauma** [AtomicGameEngine/AtomicGameEngine#75]
When I start the Editor on linux ( Linux 3.19.0-15-lowlatency #15-Ubuntu SMP PREEMPT Fri Apr 17 00:03:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux ) it looks like it uses of 50 % of both my cores with no project loaded. It never recovers until I exit the IDE. Since it doesn't peg both cores I can still work, but it seems suspicious. I have to dig deeper. But here are some screenshots of the issue.
![screenshot_2015-05-01_12-55-04](https://cloud.githubusercontent.com/assets/943437/7433664/b63ac440-f001-11e4-8b4b-d35815c27e3b.png)
![screenshot_2015-05-01_12-42-59](https://cloud.githubusercontent.com/assets/943437/7433669/bddc778e-f001-11e4-9611-bb770e77a7f2.png)
![screenshot_2015-05-01_12-44-35](https://cloud.githubusercontent.com/assets/943437/7433677/c3a4c996-f001-11e4-8249-1812eca279bb.png)

**Comment by JoshEngebretson** [comment]
Great report, I can't help to think this has something to do with renderering, though it could have something to do with filewatching (Atomic automatically reloads files like images when they change).  A profile could be good.

For reference, Windows uses 0.1% and 35.1MB ram in release:

![windowscpu](https://cloud.githubusercontent.com/assets/376203/7441524/5a059002-f0a0-11e4-92ac-826d8909b4f1.jpg)

**Comment by CTrauma** [comment]
I'll see what I can do to profile, maybe throw it at Callgrind to see what kind of stats I can collect and where time is being spent. Would have made an attempt today, but I just upgraded to Ubuntu Vivid and as usual the proprietary nvidia drivers are giving me no love as well as some systemd issues. I think I have that all worked out so I'll take a run at profiling an idle editor asap. I'll post results here..
When building the AtomicEditor on linux are there any flags I might need to set that effect performance?  All I did to build was run: cmake -DLINUX:BOOL=ON ../AtomicGameEngine & make
